File: .tmp_c4000/batch_04_go/3452_sum_of_good_numbers.go
// LeetCode 3452 - Sum of Good Numbers
// https://leetcode.com/problems/sum-of-good-numbers/

func sumOfGoodNumbers(nums []int, k int) int {
	ans := 0
	n := len(nums)
	for i, x := range nums {
		good := true
		if i-k >= 0 && x <= nums[i-k] {
			good = false
		}
		if i+k < n && x <= nums[i+k] {
			good = false
		}
		if good {
			ans += x
		}
	}
	return ans
}

File: .tmp_py4013/batch_13_dump/3452_sum_of_good_numbers.js
// LeetCode 3452 - Sum of Good Numbers
// https://leetcode.com/problems/sum-of-good-numbers/

var sumOfGoodNumbers = function(nums, k) {
    let ans = 0;
    const n = nums.length;
    for (let i = 0; i < n; i++) {
        const x = nums[i];
        let good = true;
        if (i - k >= 0 && x <= nums[i - k]) good = false;
        if (i + k < n && x <= nums[i + k]) good = false;
        if (good) ans += x;
    }
    return ans;
};
